connector/connect/src/main/scala/org/apache/spark/sql/connect/dsl/package.scala

```diff
@@ -44,7 +45,20 @@ package object dsl {
     implicit class DslExpression(val expr: proto.Expression) {
       def as(alias: String): proto.Expression = proto.Expression.newBuilder().setAlias(
         proto.Expression.Alias.newBuilder().setName(alias).setExpr(expr)).build()
+
+      def < (other: proto.Expression): proto.Expression =
+        proto.Expression.newBuilder().setUnresolvedFunction(
+          proto.Expression.UnresolvedFunction.newBuilder()
+            .addParts("<")
```

Comment:
does this actually work and we resolve '<' magically as an unresolved binary function?

Reply:
similarly would this work for `<=` or `=`?

Reply:
Yes
https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala#L764-L771